v1.1/admin_ui_runtime_dashboard.md

```diff
@@ -0,0 +1,65 @@
+---
+title: Runtime Dashboard
+toc: false
+feedback: false
+---
+
+CockroachDB’s Admin UI enables you to monitor runtime metrics such as the Node Count, CPU Time, and Memory Usage for your cluster.
+
+<div id="toc"></div>
+
+To view the Runtime metrics for your cluster, [access the Admin UI](explore-the-admin-ui.html#access-the-admin-ui). From the Dashboard drop-down box, select **Runtime**.
+
+#### Cluster and Node View for the Time-Series Graphs
+By default, the time series panel displays the cluster view, which shows the metrics for the entire cluster. For every time series data, the visualization for the entire cluster is the sum of the metric for each node. 
+
+To access the node view that shows the metrics for an individual node, select the node from the **Graph** drop-down list.
+
+<img src="{{ 'images/admin_ui_select_node.png' | relative_url }}" alt="CockroachDB Admin UI" style="border:1px solid #eee;max-width:40%"/>
+
+The Runtime dashboard displays the following time-series graphs:
+
+## Node Count
+<img src="{{ 'images/admin_ui_node_count.png' | relative_url }}" alt="CockroachDB Admin UI Node Count" style="border:1px solid #eee;max-width:100%" />
+
+In the node view as well as the cluster view, the graph displays the number of live nodes in the cluster.
+
+A dip in the graph indicates decommissioned nodes, dead nodes, or nodes that are not responding. To troubleshoot the dip in the graph, refer to the [Summary panel](explore-the-admin-ui.html#summary-panel).
+
+## Memory Usage
+<img src="{{ 'images/admin_ui_memory_usage.png' | relative_url }}" alt="CockroachDB Admin UI Memory Usage" style="border:1px solid #eee;max-width:100%" />
+
+In the node view, the graph displays the memory in use for the selected node.
+
+In the cluster view, the graph displays the memory in use across all nodes in the cluster.
+
+On hovering over the graph, the values for the following metrics are displayed:
```

Comment:
All these mentions of hovering seem strange, since the same metrics are displayed regardless of hover state.

Reply:
@couchand What I wanted to point out was that you can't see the values until you hover. Is that not getting across?

Reply:
I see what you're saying.  Yep, I do agree that's useful to point out.  Maybe at the start of the overview page, where you talk generally about the time series charts, there could be a few sentences about the details you can see on hover.